service/etcd/etcdClient.go

```diff
@@ -25,30 +24,67 @@ type EtcdClient struct {
 	leaseResponse *clientv3.LeaseGrantResponse
 }
 
+func getEtcdEndPoint() string {
```

Comment:
This code is complete duplicate of code in `operator/etcd/etcdClient.go`!

Reply:
Yes Rahul. But both operator and service are all together different sources and builds. Hence duplicated twice. If duplication needs to be avoided, then we need to place them under KVMService and use them.